app/Services/Model/Imp/SummitTicketTypeService.php

```diff
@@ -352,6 +372,23 @@ public function getAllowedTicketTypes(Summit $summit, Member $member): array
 
             $all_ticket_types = [];
 
+            $promocode = null;
+
+            if (!is_null($promocode_code)) {
+                $promocode = $summit->getPromoCodeByCode($promocode_code);
+
+                if (is_null($promocode)) {
+                    throw new EntityNotFoundException(sprintf("Promo code %s does not belongs to summit id %s.", $promocode_code, $summit->getId()));
```

Comment:
dont throw this bc the endpoint should always return tickets types , and we are opening a attack surface just dont apply it

Reply:
done